### PR TITLE
Password policy next to each other 

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
image

## Related Issue

- Fixes https://github.com/owncloud/web/issues/11142

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by AlexAndBear** [comment]
We decided to omit the max len rule, should be documented instead

File: packages/design-system/src/components/_OcTextInputPassword/_OcTextInputPassword.vue [hunk 1]
```diff
@@ -51,23 +50,27 @@
...
-          :variation="testedRule.verified ? 'success' : 'danger'"
-        />
-        <span
-          :class="[
-            { 'oc-text-input-success': testedRule.verified },
-            { 'oc-text-input-danger': !testedRule.verified }
-          ]"
-          v-text="getPasswordPolicyRuleMessage(testedRule)"
-        ></span>
+      <div class="oc-flex oc-text-input-password-policy-rule-wrapper">
```
Comment (kulmann) [inline packages/design-system/src/components/_OcTextInputPassword/_OcTextInputPassword.vue:53]:
```suggestion
      <div class="oc-text-input-password-policy-rule-wrapper">
```
You already gave the wrapper class `display: flex;` so the `oc-flex` should not be needed.

File: docker-compose.yml [hunk 2]
```diff
@@ -7,11 +7,6 @@ x-ocis-server: &ocis-service
     OCIS_LOG_LEVEL: '${OCIS_LOG_LEVEL:-error}'
     OCIS_LOG_PRETTY: '${OCIS_LOG_PRETTY:-true}'
     OCIS_LOG_COLOR: '${OCIS_LOG_COLOR:-true}'
-    OCIS_PASSWORD_POLICY_MIN_CHARACTERS: '3'
```
Comment (kulmann) [inline docker-compose.yml:10]: Reminder to re-add the password policy for the dev stack :-)

**Review by kulmann — APPROVED** [review]
Damn fine 😍

File: packages/design-system/src/components/_OcTextInputPassword/_OcTextInputPassword.vue (hunk 1 above)
Comment (AlexAndBear) [reply packages/design-system/src/components/_OcTextInputPassword/_OcTextInputPassword.vue:53]: hm this is quite needed as we need to apply     
```scss
flex-direction: row;
flex-wrap: wrap;
``` 

I removed the wrapping flexbox

**Pull request merged**